backend/src/routes/club.js

```diff
@@ -37,24 +38,111 @@ router.get("/:id", async (req, res) => {
     }
 });
 
+
+
+
+
 // Get the news for all the teams in a club
 
 router.get("/:id/news", async (req, res) => {
     try {
         const conn = await pool.getConnection();
         const news = await conn.query(`
-            SELECT title, content, published, personId, Person.firstName, Person.lastName, Team.teamName FROM Team
+            SELECT News.id,title, content, published, personId, Person.firstName, Person.lastName, Team.teamName FROM Team
             JOIN News ON Team.id = News.teamId
             JOIN Person ON Person.id = News.personId
             WHERE Team.clubId = ?
             ORDER BY published DESC
         `, [req.params.id]);
+        console.log(news)
         res.json(news);
     } catch (err) {
+        console.log(err)
         res.status(500).json({ message: err.message });
     }
 });
 
+// get specific news
+
+router.get("/:id/news/:newsId", async (req, res) => {
+  console.log("SPECIFIC NEWS ------------------");
+  try {
+    const conn = await pool.getConnection();
+    const news = await conn.query(`
+      SELECT title, content, published, personId, Person.firstName, Person.lastName, Team.teamName FROM Team
+      JOIN News ON Team.id = News.teamId
+      JOIN Person ON Person.id = News.personId
+      WHERE Team.clubId = ? AND News.id = ?
+      ORDER BY published DESC
+    `, [req.params.id, req.params.newsId]);
+    res.json(news);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+  //create news 
+  
+  router.post("/:id/news", async (req, res) => {
```

Comment:
This should be in the teams route? A team creates news, not the club. The club shows all the team news with teams connected to the club.